lib/modules/reducer.js

```diff
@@ -14,25 +14,23 @@ const DEFAULT = {
 };
 
 export default (state=DEFAULT, action={}) => {
-  switch(action.type) {
+  switch (action.type) {
     case TOGGLE_TOOLTIP: {
       const { tooltipId, target } = action.payload;
 
-      return (tooltipId && target)
-        ? {
-            ...state,
-            tooltip: {
-              target,
-              id: tooltipId,
-            },
-          }
-        : {
-            ...state,
-            tooltip: {
-              target: null,
-              id: null,
-            },
-          }
+      return (tooltipId && target) ? {
```

Comment:
same with whatever rule prompted this change. in general, all of our ternaries have looked like:

```es6
thing
  ? foo
  : bar;
```

If that breaks our linting rules in certain ways, we need to revisit our rules.